src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Although it may seem like a convenient shortcut to just import `.*`, this violates our coding standards (which you should adhere to) and is bad practice especially for big projects where a package can contain many other classes which you do not need to use here.